lang/po/Firmware_sv.po

```diff
@@ -18,482 +18,482 @@ msgstr ""
 # MSG_IR_03_OR_OLDER c=18
 #: messages.c:164
 msgid " 0.3 or older"
-msgstr ""
+msgstr " 0.3 el äldre"
 
 # MSG_FS_V_03_OR_OLDER c=18
 #: Marlin_main.cpp:9887
 msgid "FS v0.3 or older"
-msgstr ""
+msgstr "FS v0.3 el äldre"
 
 # MSG_IR_04_OR_NEWER c=18
 #: messages.c:163
 msgid " 0.4 or newer"
-msgstr ""
+msgstr " 0.4 el nyare"
 
 # MSG_FS_V_04_OR_NEWER c=18
 #: Marlin_main.cpp:9886
 msgid "FS v0.4 or newer"
-msgstr ""
+msgstr "FS v0.4 el nyare"
 
 # MSG_IR_UNKNOWN c=18
 #: messages.c:165
 msgid "unknown state"
-msgstr ""
+msgstr "okänt tillstand"
 
 # MSG_MEASURED_OFFSET c=20
 #: ultralcd.cpp:2913
 msgid "[0;0] point offset"
-msgstr ""
+msgstr "[0;0] punktförskjutn"
 
 # MSG_CRASH_DET_ONLY_IN_NORMAL c=20 r=4
 #: 
 msgid "Crash detection can\x0abe turned on only in\x0aNormal mode"
-msgstr ""
+msgstr "Krockdetekt kan\x0aendast aktiveras i\x0anormalt läge"
 
 # MSG_CRASH_DET_STEALTH_FORCE_OFF c=20 r=4
 #: 
 msgid "WARNING:\x0aCrash detection\x0adisabled in\x0aStealth mode"
-msgstr ""
+msgstr "VARNING:\x0aKrockdetekt\x0ainaktiverad i\x0atyst-läge"
 
 # MSG_BABYSTEPPING_Z c=15
 #: ultralcd.cpp:3034
 msgid "Adjusting Z:"
-msgstr ""
+msgstr "Justerar Z:"
 
 # MSG_SELFTEST_CHECK_ALLCORRECT c=20
 #: ultralcd.cpp:8411
 msgid "All correct"
-msgstr ""
+msgstr "Allt korrekt"
 
 # MSG_WIZARD_DONE c=20 r=3
 #: messages.c:118
 msgid "All is done. Happy printing!"
-msgstr ""
+msgstr "Allt är klart. God utskrift!"
 
 # MSG_AMBIENT c=14
 #: ultralcd.cpp:1727
 msgid "Ambient"
-msgstr ""
+msgstr "Omgivande"
 
 # MSG_AUTO c=6
 #: messages.c:160
 msgid "Auto"
-msgstr ""
+msgstr "Auto"
 
 # MSG_PRESS c=20 r=2
 #: ultralcd.cpp:2485
 msgid "and press the knob"
-msgstr ""
+msgstr "och tryck pa knappen"
 
 # MSG_CONFIRM_CARRIAGE_AT_THE_TOP c=20 r=2
 #: ultralcd.cpp:3355
 msgid "Are left and right Z~carriages all up?"
-msgstr ""
+msgstr "är bada Z-vagnarna helt uppe?"
 
 # MSG_AUTO_HOME c=18
 #: messages.c:11
 msgid "Auto home"
-msgstr ""
+msgstr "Auto hem"
 
 # MSG_AUTOLOAD_FILAMENT c=18
 #: ultralcd.cpp:6653
 msgid "AutoLoad filament"
-msgstr ""
+msgstr "Autoladda filament"
 
 # MSG_AUTOLOADING_ONLY_IF_FSENS_ON c=20 r=4
 #: ultralcd.cpp:4317
 msgid "Autoloading filament available only when filament sensor is turned on..."
-msgstr ""
+msgstr "Autoladdning av fil. är endast möjligt när fil. sensorn är aktiv..."
 
 # MSG_AUTOLOADING_ENABLED c=20 r=4
 #: ultralcd.cpp:2648
 msgid "Autoloading filament is active, just press the knob and insert filament..."
-msgstr ""
+msgstr "Autoladdning filament är aktiv, tryck pa knappen och sätt i filament..."
 
 # MSG_SELFTEST_AXIS_LENGTH c=20
 #: ultralcd.cpp:8094
 msgid "Axis length"
-msgstr ""
+msgstr "Axellängd"
 
 # MSG_SELFTEST_AXIS c=16
 #: ultralcd.cpp:8095
 msgid "Axis"
-msgstr ""
+msgstr "Axel"
```

Comment:
Google translate gives me some weird responses, but as long as it is Axis, like an axis of a coordinate system and not an "axle" like you find on a car, this is correct. Not sure of the swedish here.